TheMET/TheMET/FailedToLoadView/FailedToLoadView.xib

```diff
@@ -0,0 +1,62 @@
+                    <rect key="frame" x="121.66666666666669" y="179" width="150" height="150"/>
+                    <constraints>
+                        <constraint firstAttribute="height" constant="150" id="4AD-P6-1zC"/>
+                        <constraint firstAttribute="width" constant="150" id="sya-My-l2x"/>
+                    </constraints>
+                </imageView>
+                <label opaque="NO" userInteractionEnabled="NO" contentMode="left" horizontalHuggingPriority="251" verticalHuggingPriority="251" text="Label" textAlignment="center" lineBreakMode="tailTruncation" numberOfLines="0" baselineAdjustment="alignBaselines" adjustsFontSizeToFit="NO" translatesAutoresizingMaskIntoConstraints="NO" id="zpv-wu-J7I">
+                    <rect key="frame" x="50" y="429" width="293" height="61"/>
+                    <fontDescription key="fontDescription" type="system" pointSize="17"/>
+                    <nil key="textColor"/>
+                    <nil key="highlightedColor"/>
+                </label>
+                <button opaque="NO" contentMode="scaleToFill" contentHorizontalAlignment="center" contentVerticalAlignment="center" buttonType="system" lineBreakMode="middleTruncation" translatesAutoresizingMaskIntoConstraints="NO" id="bbX-wp-Twx">
+                    <rect key="frame" x="50" y="590" width="293" height="60"/>
+                    <constraints>
+                        <constraint firstAttribute="height" constant="60" id="Inh-pG-J62"/>
+                    </constraints>
+                    <state key="normal" title="Button"/>
+                    <buttonConfiguration key="configuration" style="plain" title="Button"/>
+                </button>
+            </subviews>
+            <viewLayoutGuide key="safeArea" id="vUN-kp-3ea"/>
+            <color key="backgroundColor" white="0.0" alpha="0.0" colorSpace="custom" customColorSpace="genericGamma22GrayColorSpace"/>
+            <constraints>
+                <constraint firstItem="vUN-kp-3ea" firstAttribute="trailing" secondItem="bbX-wp-Twx" secondAttribute="trailing" constant="50" id="0Px-Y4-Fph"/>
+                <constraint firstItem="ebe-23-fjT" firstAttribute="top" secondItem="vUN-kp-3ea" secondAttribute="top" constant="120" id="Hta-8z-i5m"/>
+                <constraint firstItem="vUN-kp-3ea" firstAttribute="trailing" secondItem="zpv-wu-J7I" secondAttribute="trailing" constant="50" id="NBs-nV-3Wf"/>
+                <constraint firstItem="bbX-wp-Twx" firstAttribute="leading" secondItem="vUN-kp-3ea" secondAttribute="leading" constant="50" id="UZT-5i-IUg"/>
+                <constraint firstItem="zpv-wu-J7I" firstAttribute="centerX" secondItem="vUN-kp-3ea" secondAttribute="centerX" id="agr-fH-4dq"/>
+                <constraint firstItem="bbX-wp-Twx" firstAttribute="centerX" secondItem="vUN-kp-3ea" secondAttribute="centerX" id="ajw-Ep-coN"/>
```

Comment:
Here and in other XIB:
There is no need to center button and title horizontally. They have equal leading and trailing insets. It means they are already centered

Reply:
fixed